### PR TITLE
Michal

### PR DESCRIPTION
**Pull request by michalz92**

### CODE REVIEW

File: grizly/etl.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
     pass
 
 
-def to_csv(qf,csv_path, sql, engine, sep='\t', chunksize=None, cursor=None):
+def to_csv(qf,csv_path, sql, engine=None, sep='\t', chunksize=None, cursor=None):
```
Comment (malinaa96) [inline grizly/etl.py]: Do we even need an engine string here? If we have the qf then we can read an engine string from it right?

File: grizly/utils.py [hunk 2]
```diff
@@ -138,7 +138,7 @@ def get_sfdc_columns(table, columns=None, column_types=True):
         raise NotImplementedError("Retrieving columns only is currently not supported")
 
 
-def get_denodo_columns(schema, table, column_types=False, date_format="DATE"):
+def get_denodo_columns(schema, table, column_types=False, columns=None, date_format="DATE"):
```
Comment (malinaa96) [inline grizly/utils.py]: Nice feature, I would also add some kind of warning when the user gives some columns in `columns` argument that are not found in Denodo table.

File: grizly/etl.py (hunk 1 above)
Comment (michalz92) [reply grizly/etl.py]: Yeah, we can read from QF.

File: grizly/utils.py (hunk 2 above)
Comment (michalz92) [reply grizly/utils.py]: Open an issue. :p

Comment (malinaa96) [reply grizly/utils.py]: done https://github.com/kfk/grizly/issues/206

**Pull request merged**